carsons/carsons.py

```diff
@@ -19,13 +18,43 @@ def convert_geometric_model(geometric_model):
 
 
 def perform_kron_reduction(z_primitive):
-    z_phase_phase = z_primitive[0:3, 0:3]
-    z_phase_neutral = z_primitive[0:3, 3:]
-    z_neutral_phase = z_primitive[3:, 0:3]
-    z_neutral_neutral = z_primitive[3:, 3:]
-    z_neutral_part = z_phase_neutral * inv(z_neutral_neutral) * z_neutral_phase
-    z_abc = z_phase_phase - z_neutral_part
-    return z_abc
+    """ Reduces the primitive impedance matrix to an equivalent impedance
+        matrix.
+
+        We break z_primative up into four quadrants as follows:
+
+              Žpp = [Žaa, Žab, Žac]   Žpn = [Žan]
```

Comment:
Do all the zeds have a weird inverted "circumflex" accent on them or is that just how the characters are rendering on my computer?

Reply:
They do have a circumflex. I did this to show that the impedance values in z_pp, z_pn etc. can't be used interchangeably with the values in z_abc, because they don't account for the neutral wires. It matches the notation of the source i cited in the reference document

Reply:
Oo. I found a unicode Ẑ, which is better